src/pybind/mgr/cephadm/module.py

```diff
@@ -282,7 +282,7 @@ class CephadmOrchestrator(orchestrator.Orchestrator, MgrModule,
             'registry_url',
             type='str',
             default=None,
-            desc='Custom repository url'
+            desc='Registry url for login purposes. This is not the default registry'
```

Comment:
Can we also change the "registry_username" and "registry_password" descs to match. I.e. "Registry username for login purposes . . ."